modules/arm_plugin/src/arm_converter/arm_converter.cpp

```diff
@@ -174,6 +174,7 @@ Converter::Converter(const std::shared_ptr<const ngraph::Function> function, boo
         Register<opset::IDFT>();
         Register<opset::FakeQuantize>();
         Register<opset::Split>();
+        Register<ngraph::op::v8::Gather>();
```

Comment:
Should we update registration for `opset::Gather` to `v7::Gather` as well to avoid conflict upon migration to v8 as common opset?

Reply:
@terfendail done